modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/freelist/FreeListImpl.java

```diff
@@ -863,7 +816,7 @@ public long recycledPagesCount() throws IgniteInternalCheckedException {
 
     @Override
     public void saveMetadata() throws IgniteInternalCheckedException {
-        saveMetadata(statHolder);
+        super.saveMetadata();
```

Comment:
It seems that this override can just be removed

Reply:
Added a comment to the code